From f080c2ef542b0d3c78eb2f98a4450a8e22b471af Mon Sep 17 00:00:00 2001
From: Piotr Chmura <chmooreck@poczta.onet.pl>
Date: Mon, 31 Oct 2011 12:24:50 -0300
Subject: [media] staging: as102: Remove non-linux headers inclusion
Git-commit: f080c2ef542b0d3c78eb2f98a4450a8e22b471af
Patch-mainline: v3.2-rc2

Remove inclusion of Windows and other not linux related headers.

[snjw23@gmail.com: edited changelog, folded long line in Makefile]
Cc: Devin Heitmueller <dheitmueller@kernellabs.com>
Signed-off-by: Piotr Chmura <chmooreck@poczta.onet.pl>
Signed-off-by: Sylwester Nawrocki <snjw23@gmail.com>
Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/staging/media/as102/Makefile           |    5 +++--
 drivers/staging/media/as102/as10x_cmd.c        |   22 ----------------------
 drivers/staging/media/as102/as10x_cmd_cfg.c    |   22 ----------------------
 drivers/staging/media/as102/as10x_cmd_stream.c |   22 +---------------------
 4 files changed, 4 insertions(+), 67 deletions(-)

diff --git a/drivers/staging/media/as102/Makefile b/drivers/staging/media/as102/Makefile
index c2334c6..e7dbb6f 100644
--- a/drivers/staging/media/as102/Makefile
+++ b/drivers/staging/media/as102/Makefile
@@ -1,5 +1,6 @@
-dvb-as102-objs := as102_drv.o as102_fw.o as10x_cmd.o as10x_cmd_stream.o as102_fe.o as102_usb_drv.o as10x_cmd_cfg.o
+dvb-as102-objs := as102_drv.o as102_fw.o as10x_cmd.o as10x_cmd_stream.o \
+		as102_fe.o as102_usb_drv.o as10x_cmd_cfg.o
 
 obj-$(CONFIG_DVB_AS102) += dvb-as102.o
 
-EXTRA_CFLAGS += -DLINUX -DCONFIG_AS102_USB -Idrivers/media/dvb/dvb-core
+EXTRA_CFLAGS += -DCONFIG_AS102_USB -Idrivers/media/dvb/dvb-core
diff --git a/drivers/staging/media/as102/as10x_cmd.c b/drivers/staging/media/as102/as10x_cmd.c
index 222e703..77aae39 100644
--- a/drivers/staging/media/as102/as10x_cmd.c
+++ b/drivers/staging/media/as102/as10x_cmd.c
@@ -18,30 +18,8 @@
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
-#if defined(LINUX) && defined(__KERNEL__) /* linux kernel implementation */
 #include <linux/kernel.h>
 #include "as102_drv.h"
-#elif defined(WIN32)
-   #if defined(__BUILDMACHINE__) && (__BUILDMACHINE__ == WinDDK)
-      /* win32 ddk implementation */
-      #include "wdm.h"
-      #include "Device.h"
-      #include "endian_mgmt.h" /* FIXME */
-   #else /* win32 sdk implementation */
-      #include <windows.h>
-      #include "types.h"
-      #include "util.h"
-      #include "as10x_handle.h"
-      #include "endian_mgmt.h"
-   #endif
-#else /* all other cases */
-   #include <string.h>
-   #include "types.h"
-   #include "util.h"
-   #include "as10x_handle.h"
-   #include "endian_mgmt.h" /* FIXME */
-#endif /* __KERNEL__ */
-
 #include "as10x_types.h"
 #include "as10x_cmd.h"
 
diff --git a/drivers/staging/media/as102/as10x_cmd_cfg.c b/drivers/staging/media/as102/as10x_cmd_cfg.c
index 0635797..dca2cbf 100644
--- a/drivers/staging/media/as102/as10x_cmd_cfg.c
+++ b/drivers/staging/media/as102/as10x_cmd_cfg.c
@@ -17,30 +17,8 @@
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
-#if defined(LINUX) && defined(__KERNEL__) /* linux kernel implementation */
 #include <linux/kernel.h>
 #include "as102_drv.h"
-#elif defined(WIN32)
-   #if defined(__BUILDMACHINE__) && (__BUILDMACHINE__ == WinDDK)
-      /* win32 ddk implementation */
-      #include "wdm.h"
-      #include "Device.h"
-      #include "endian_mgmt.h" /* FIXME */
-   #else /* win32 sdk implementation */
-      #include <windows.h>
-      #include "types.h"
-      #include "util.h"
-      #include "as10x_handle.h"
-      #include "endian_mgmt.h"
-   #endif
-#else /* all other cases */
-   #include <string.h>
-   #include "types.h"
-   #include "util.h"
-   #include "as10x_handle.h"
-   #include "endian_mgmt.h" /* FIXME */
-#endif /* __KERNEL__ */
-
 #include "as10x_types.h"
 #include "as10x_cmd.h"
 
diff --git a/drivers/staging/media/as102/as10x_cmd_stream.c b/drivers/staging/media/as102/as10x_cmd_stream.c
index b5e6254..e8c668c 100644
--- a/drivers/staging/media/as102/as10x_cmd_stream.c
+++ b/drivers/staging/media/as102/as10x_cmd_stream.c
@@ -16,29 +16,9 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
-#if defined(LINUX) && defined(__KERNEL__) /* linux kernel implementation */
+
 #include <linux/kernel.h>
 #include "as102_drv.h"
-#elif defined(WIN32)
-    #if defined(DDK) /* win32 ddk implementation */
-	#include "wdm.h"
-	#include "Device.h"
-	#include "endian_mgmt.h" /* FIXME */
-    #else /* win32 sdk implementation */
-	#include <windows.h>
-	#include "types.h"
-	#include "util.h"
-	#include "as10x_handle.h"
-	#include "endian_mgmt.h"
-    #endif
-#else /* all other cases */
-    #include <string.h>
-    #include "types.h"
-    #include "util.h"
-    #include "as10x_handle.h"
-    #include "endian_mgmt.h" /* FIXME */
-#endif /* __KERNEL__ */
-
 #include "as10x_cmd.h"
 
 
-- 
1.7.7.3

